README.md

```
@@ -9,12 +9,16 @@
    1. BETTERAUTH_SECRET - ランダムな英数字の羅列を挿入
    2. GOOGLE_CLIENT_ID / SECRET - Google API PlatformのClient ID / Secretです。詳しくは後述します。
    3. BETTER_AUTH_URL - 動かすURLを入力
-   4. DATABASE_URL - DBのURI。デフォルトでdocker-compose.yamlを動かした場合のものを入力済み
-   5. SMTP*HOST / SMT*... - メールの設定。パスワードリセットやメール認証で用いられます。
-   6. RAKUTEN_APP_ID / KEY - 楽天APIの設定です。これを設定することにより、ISBNコードの検索が容易になります。詳しくは後述します。
-5. `prisma generate`
-6. `prisma migrate dev`
-7. `bun dev`
+   4. 各環境変数を埋める
+    1. BETTERAUTH_SECRET - ランダムな英数字の羅列を挿入
```

markdownlint-cli2 (0.22.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 3; Actual: 4

(MD005, list-indent)